### DISCUSSION TITLE
Nuevo Integrante

### DISCUSSION THREAD

**Discussion by gonaiot** [ISPC-TST-Electronica-Microcontrolada/Grupo5#20] [Announcements]
Estimados, en la reorganización de grupos hemos completado su grupo con la siguiente alumna: 

Nombre: Silvana del Pilar Barea         
e-mail: silvanahousseini@gmail.com
github: gitdesil

**Comment by Mauro-Martinez** [comment]
Buenas profee! Bienvenid@s a los nuevos integrantes, adjunto enlace a nuestro grupo de whatsapp por si se quieren unir https://chat.whatsapp.com/HFTZfOFkyzxLXBhIsVm7oe
